src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -330,6 +333,7 @@ private void loadConfig() {
         try (InputStreamReader reader = new InputStreamReader(new FileInputStream(config), StandardCharsets.UTF_8)) {
             this.config = GSON.fromJson(reader, Config.class);
             if (this.config == null) this.config = new Config();
+            this.initializeVirtualConfigs();
```

Comment:
```suggestion
```
config reading shouldn't be modifying the config. Also, just don't store the VALUE in config, save it in wherever refresh is being handled, as a code-variable value.